lib/utils.js

```diff
@@ -807,6 +807,10 @@ function retrieveEJSON() {
   return EJSON;
 }
 
+/**
+ * Retrieves pure js or native bson library
+ * @returns {import('bson')}
```

Comment:
this `@returns` block looks funky, should it just say it returns an `Object` or is this some new jsdoc/tsdoc fu I'm not aware of?

Reply:
A little preview of the type of thing we can do in JSDoc to get typescript checks for us, this will tell vscode to pull in the `@types/bson` for auto suggesting whatever this function returns, I put it here while I was checking stuff and it slipped through. I'll save this for TS stuff.